HISTORY.md

```diff
@@ -1,4 +1,4 @@
-# In Progress
+# TileDB v2.2.8 Release Notes
```

Comment:
@Shelnutt2 We won't need to maintain the empty "In Progress" section with the new way of generating history.